.env

```diff
@@ -1 +1,2 @@
 SECRET_KEY_BASE=123
+BB_SECRET='this is a test secret'
```

Comment:
Oh, I don't think this file should be checked in. The `.env.test` file can be, but the `.env` file is for our own personal use.